Edit Command - modifying grade beyond bounds throws runtime error and no error message

The `edit /id A0237297R /grade 1000` command to test the bounds of a grade does not show any error messages or indication when it is entered. 

![Image](https://github.com/user-attachments/assets/70f5e90b-86c8-4bc0-b530-15c7d6a27f3b)


Runtime error: 
```
INFO: ----------------[USER COMMAND][edit /id A0237297R /grade 1000]
Exception in thread "JavaFX Application Thread" java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
        at javafx.fxml@21.0.6/javafx.fxml.FXMLLoader$MethodHandler.invoke(FXMLLoader.java:1858)
        at javafx.fxml@21.0.6/javafx.fxml.FXMLLoader$ControllerMethodEventHandler.handle(FXMLLoader.java:1726)
        at javafx.base@21.0.6/com.sun.javafx.event.CompositeEventHandler.dispatchBubblingEvent(CompositeEventHandler.java:86)
        at javafx.base@21.0.6/com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(EventHandlerManager.java:232)
        at javafx.base@21.0.6/com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(EventHandlerManager.java:189)
        at javafx.base@21.0.6/com.sun.javafx.event.CompositeEventDispatcher.dispatchBubblingEvent(CompositeEventDispatcher.java:59)
        at javafx.base@21.0.6/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:58)
        at javafx.base@21.0.6/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
        at javafx.base@21.0.6/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:56)
        at javafx.base@21.0.6/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
        at javafx.base@21.0.6/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:56)
        at javafx.base@21.0.6/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
        at javafx.base@21.0.6/com.sun.javafx.event.EventUtil.fireEventImpl(EventUtil.java:74)
        at javafx.base@21.0.6/com.sun.javafx.event.EventUtil.fireEvent(EventUtil.java:49)
        at javafx.base@21.0.6/javafx.event.Event.fireEvent(Event.java:198)
        at javafx.graphics@21.0.6/javafx.scene.Node.fireEvent(Node.java:8875)
        at javafx.controls@21.0.6/com.sun.javafx.scene.control.behavior.TextFieldBehavior.fire(TextFieldBehavior.java:161)
        at javafx.controls@21.0.6/com.sun.javafx.scene.control.behavior.TextInputControlBehavior.lambda$keyMapping$62(TextInputControlBehavior.java:344)
        at javafx.controls@21.0.6/com.sun.javafx.scene.control.inputmap.InputMap.handle(InputMap.java:274)
        at javafx.base@21.0.6/com.sun.javafx.event.CompositeEventHandler$NormalEventHandlerRecord.handleBubblingEvent(CompositeEventHandler.java:247)
        at javafx.base@21.0.6/com.sun.javafx.event.CompositeEventHandler.dispatchBubblingEvent(CompositeEventHandler.java:80)
        at javafx.base@21.0.6/com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(EventHandlerManager.java:232)
        at javafx.base@21.0.6/com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(EventHandlerManager.java:189)
        at javafx.base@21.0.6/com.sun.javafx.event.CompositeEventDispatcher.dispatchBubblingEvent(CompositeEventDispatcher.java:59)
        at javafx.base@21.0.6/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:58)
        at javafx.base@21.0.6/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
        at javafx.base@21.0.6/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:56)
        at javafx.base@21.0.6/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
        at javafx.base@21.0.6/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:56)
        at javafx.base@21.0.6/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
        at javafx.base@21.0.6/com.sun.javafx.event.EventUtil.fireEventImpl(EventUtil.java:74)
        at javafx.base@21.0.6/com.sun.javafx.event.EventUtil.fireEvent(EventUtil.java:54)
        at javafx.base@21.0.6/javafx.event.Event.fireEvent(Event.java:198)
        at javafx.graphics@21.0.6/javafx.scene.Scene.processKeyEvent(Scene.java:2194)
        at javafx.graphics@21.0.6/javafx.scene.Scene$ScenePeerListener.keyEvent(Scene.java:2715)
        at javafx.graphics@21.0.6/com.sun.javafx.tk.quantum.GlassViewEventHandler$KeyEventNotification.run(GlassViewEventHandler.java:218)
        at javafx.graphics@21.0.6/com.sun.javafx.tk.quantum.GlassViewEventHandler$KeyEventNotification.run(GlassViewEventHandler.java:150)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
        at javafx.graphics@21.0.6/com.sun.javafx.tk.quantum.GlassViewEventHandler.lambda$handleKeyEvent$1(GlassViewEventHandler.java:250)
        at javafx.graphics@21.0.6/com.sun.javafx.tk.quantum.QuantumToolkit.runWithoutRenderLock(QuantumToolkit.java:424)
        at javafx.graphics@21.0.6/com.sun.javafx.tk.quantum.GlassViewEventHandler.handleKeyEvent(GlassViewEventHandler.java:249)
        at javafx.graphics@21.0.6/com.sun.glass.ui.View.handleKeyEvent(View.java:542)
        at javafx.graphics@21.0.6/com.sun.glass.ui.View.notifyKey(View.java:966)
        at javafx.graphics@21.0.6/com.sun.glass.ui.gtk.GtkApplication._runLoop(Native Method)
        at javafx.graphics@21.0.6/com.sun.glass.ui.gtk.GtkApplication.lambda$runLoop$10(GtkApplication.java:263)
        at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: java.lang.reflect.InvocationTargetException
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:568)
        at com.sun.javafx.reflect.Trampoline.invoke(MethodUtil.java:72)
        at jdk.internal.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:568)
        at javafx.base@21.0.6/com.sun.javafx.reflect.MethodUtil.invoke(MethodUtil.java:270)
        at javafx.fxml@21.0.6/com.sun.javafx.fxml.MethodHelper.invoke(MethodHelper.java:84)
        at javafx.fxml@21.0.6/javafx.fxml.FXMLLoader$MethodHandler.invoke(FXMLLoader.java:1855)
        ... 45 more
Caused by: java.lang.IllegalArgumentException: Grades should only be an integer between 0 and 100, inclusive
        at seedu.address.commons.util.AppUtil.checkArgument(AppUtil.java:39)
        at seedu.address.model.person.Grade.<init>(Grade.java:32)
        at seedu.address.logic.parser.EditCommandParser.parse(EditCommandParser.java:79)
        at seedu.address.logic.parser.AddressBookParser.parseCommand(AddressBookParser.java:64)
        at seedu.address.logic.LogicManager.execute(LogicManager.java:53)
        at seedu.address.ui.MainWindow.executeCommand(MainWindow.java:208)
        at seedu.address.ui.CommandBox.handleCommandEntered(CommandBox.java:45)
        ... 56 more
```

--- a/src/main/java/seedu/address/model/person/Grade.java
+++ b/src/main/java/seedu/address/model/person/Grade.java
@@ -40,7 +40,7 @@ public class Grade {
 
             int parsedGrade = Integer.parseInt(grade);
 
-            if (parsedGrade == -1) {
+            if (parsedGrade < 0 || parsedGrade > 100) {
                 throw new ParseException(MESSAGE_CONSTRAINTS);
             }
 

